fiftyone/factory/repos/delegated_operation_doc.py

```
@@ -92,7 +92,7 @@ def from_pymongo(self, doc: dict):
         self.updated_at = doc.get("updated_at", None)
 
         # grouped fields
-        self.group_id = doc.get("group_id", None)
+        self.parent_id = doc.get("parent_id", None)
```

Ruff (0.12.2)

Use `doc.get("parent_id")` instead of `doc.get("parent_id", None)`

Replace `doc.get("parent_id", None)` with `doc.get("parent_id")`

(SIM910)